apps/web/src/features/user/dashboard/components/dashboard-activity.css

```
@@ -1,11 +1,11 @@
 .dashboard-activity-content {
   --activity-cell: max(
     20px,
-    calc((100cqw - 56px - var(--activity-weeks) * 4px) / var(--activity-weeks))
+    calc((100cqw - 58px - var(--activity-weeks) * 4px) / var(--activity-weeks))
   );
-  height: calc(7 * var(--activity-cell) + 160px);
+  height: calc(7 * var(--activity-cell) + 112px);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)